### DISCUSSION TITLE
LCD Display not working--Octoprint

### DISCUSSION THREAD

**Discussion by HermanGaviriaMarin** [Jyers/Marlin#972] [Q&A] [answered]
I recently installed a Raspberry 4B 4GB to use Octoprint and all of the glorious plugging's. The firmware worked fine as it the PI connects to the printer without any issues, all commands are received and work while made from my computer. The only issue that I find myself facing is that the LCD display does not show the print name, the progress, or how much time is left. This of course wouldn't matter if we choose to check all of this from Octoprint, but of course having both features is better than just having one. So basically to sum up, I use Octoprint and it works but LCD display doesn't display correct info, I can see the missing information on the Octoprint tab, I can still control some things with the LCD. If this is an issue I generated how do I fix it, and if its just the price we pay for using Octoprint thats cool too just lmk.
Thanks for your time
![image](https://user-images.githubusercontent.com/84948613/120954887-2b0ab380-c71e-11eb-842c-2925d31e3cfe.png)

**Comment by smooreace** [comment]
Looks like its showing progress to me
Whats 88% and Layer 110 of 125?

**Comment by gothcha** [comment]
You need to use M73 commands to update the progress bar and remaining time. Add the post process plugin in Cura which inserts M73 for each layer

**Comment by Jyers** [comment, marked as answer]
Filename, progress, time remaining! All these and more can be yours for the low low price of checking the wiki:

https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings

**Comment by smooreace** [comment]
Too much work... its easier to ask
